app/ts/components/ConfigureFunding.tsx

```diff
@@ -127,31 +128,63 @@ const WithdrawModal = ({ display, blockInfo, signers, appSettings, provider }: {
 	}
 
 	const withdrawAmount = useComputed(() => {
-		let fee = (blockInfo.value.baseFee + blockInfo.value.priorityFee) * 11n / 10n * 21000n
+		let maxFeePerGas = getMaxBaseFeeInFutureBlock(blockInfo.value.baseFee, 20n) + blockInfo.value.priorityFee;
```

Comment:
```suggestion
		let maxFeePerGas = getMaxBaseFeeInFutureBlock(blockInfo.value.baseFee, 5n) + blockInfo.value.priorityFee;
```

Allow for max 60 seconds of worse case gas increases.